pkg/ccl/backupccl/datadriven_test.go

```diff
@@ -386,6 +410,25 @@ func TestDataDriven(t *testing.T) {
 				lastCreatedServer = name
 				return ""
 
+			case "upgrade-server":
+				server := lastCreatedServer
+				user := "root"
+
```

Comment:
keeping track of the server upgrade channel and closing it isn't strictly necessary. we could keep this channel permanently open so the test writer can upgrade the cluster several times. this isn't too important. wdyt?

Reply:
I'm okay with closing the channel at the time of final cleanup if it still allows us to upgrade using set cluster setting.

Reply:
After looking at a few other tests that use this feature, I realized we don't really need to close the channel ever. When the server spins down, everything is taken care of. Deleted some code.